requirements.txt

```diff
@@ -1,3 +1,5 @@
+pip==21.3.1
```

Comment:
Nit: pip isn't a code dependency, unless this particular version of pip is required. It is usually preferable to just upgrade the native pip (as we have to do at EROS).

Reply:
ok, I can remove it. I got a notice when using pip so I upgraded it. I know that we upgrade for EROS machines and this will be running on Jenkins ERORS.